### PR TITLE
refine darcula and darcula-solid themes

### PR DESCRIPTION
**Pull request by boofexxx**
### darcula:

Screenshot from 2023-09-27 16-21-30

### darcula-solid:

Screenshot from 2023-09-27 16-21-26

### darcula:

Screenshot from 2023-09-27 16-21-16

### darcula-solid:

Screenshot from 2023-09-27 16-21-03

### CODE REVIEW

**Comment by the-mikedavis** [comment]
\cc @nogden & @jesusmgg - what do you think?

**Comment by archseer** [comment]
I would keep the default cursor shape for secondary selections, the underline seems more like a stylistic choice

**Comment by jesusmgg** [comment]
I concur, the secondary selection color seems like an upgrade to me but let's keep the original cursor shape.

File: runtime/themes/darcula.toml [hunk 1]
```diff
@@ -8,12 +8,13 @@
...
 "ui.window" = { bg = "grey01" }
-"ui.selection" = { bg = "grey03" }
+"ui.selection" = { bg = "grey02" }
+"ui.selection.primary" = { bg = "blue" }
 "ui.statusline" = { fg = "grey04", bg = "grey02" }
 "ui.statusline.insert" = { bg = "white", fg = "grey01" }
 "ui.statusline.select" = { bg = "orange", fg = "grey01" }
 "ui.help" = { fg = "grey04", bg = "grey01" }
-"ui.cursor" = { fg = "grey04", modifiers = ["reversed"] }
+"ui.cursor" = { bg = "grey02", modifiers = ["underlined"] }
```
Comment (nogden) [inline runtime/themes/darcula.toml:17]: As discussed lets keep this as the default.

**Comment by boofexxx** [comment]
I made changes to secondary cursor because for some reasons they seem to be distracting for me

### master

image

### changed version

image

# Bug

And seems like I'm introducing a bug with these changes for **darcula-solid** because now it's not possible to say which argument we're on in documentation popups

image

With the following changes on `darcula-solid.toml` it gets better a lil bit:

```
"ui.popup" = { fg = "grey03", bg = "grey00" }
```

image

## for just darcula it looks pretty fine

### master

image

### changed version

image


# markup.raw

And if you don't mind can we change `markup.raw` in `darcula.toml` from white to purple or any other color

```
"markup.raw" = "white"
```

image


to

```
"markup.raw" = "purple"
```

image

**Comment by jesusmgg** [comment]
Everything looks great now to me. Just the PR title should be updated to reflect the more extensive changes.

**Review by nogden — APPROVED** [review]

**Review by jesusmgg — APPROVED** [review]

**Review by the-mikedavis — APPROVED** [review]

**Review by pascalkuthe — APPROVED** [review]

**Pull request merged**